### ISSUE TITLE
Duplicate Microsoft MDE PowerBI Templates?

### ISSUE THREAD

**Issue by SmittySec** [microsoft/MDE-PowerBI-Templates#10]
@YongRhee-MSFT  - I'm trying to track down firewall events, and came across two Microsoft projects for MDE Power BI templates?  

Do you know which one is actively maintained?  MDE has deleted the firewall dashboard, and this one using [Fully spelled Defender for Endpoint](https://github.com/microsoft/MicrosoftDefenderForEndpoint-PowerBI) has it up.

This might be a candidate for merging or archive one.  

Thanks, 
Rich

PS any reason why the FW dashboard was deleted in this github?

**Comment by YongRhee-MSFT** [comment]
@SmittySec , do you have Microsoft Defender for Endpoint (MDE, formerly known as Microsoft Defender Advanced Threat Protection (MDATP))?
If not, we have a new Windows Firewall report in MEM (Intune).  You could learn more about it here: https://docs.microsoft.com/en-us/mem/intune/fundamentals/whats-new#endpoint-security-firewall-reports    Thx.

**Comment by SmittySec** [comment]
Yes, currently rolling out MDE.  We confirmed with our fast track consultant from Avanade, and they were also having problems pulling the data using the Advanced Hunting Query from other tenants as well.  I confirmed we are auditing success/failures on the auditpol.  Looks like something might have changed with how that ActionType is forwarded?

Thanks,

Rich
________________________________
